### PR TITLE
APP-411: Profiling and grouping demo - description mistakes

### PR DESCRIPTION
**Pull request by am1kk**
http://jira.kaaproject.org/browse/APP-411

### CODE REVIEW

**Comment by am1kk** [comment]
@tonyshkurenko review please.

**Comment by Slash32** [comment]
@kaa-leeroy test this fully please

File: profilingdemo/resources/projects.xml [hunk 1]
```diff
@@ -1077,7 +1077,7 @@ To do this:
 </p>
 
 <p>
-	Create a group named <b>Enable vibration if supported by device</b> and set its weight to <b>120</b>.
+	Create a group named <b>Enable video if supported by device</b> and set its weight to <b>130</b>.
```
Comment (antonshkurenko) [inline profilingdemo/resources/projects.xml]: 120 instead of 130 for the video

File: profilingdemo/resources/projects.xml [hunk 2]
```diff
@@ -1089,15 +1089,15 @@ To do this:
...
+		  "videoSubscriptionActive" : true,
+		  "vibroSubscriptionActive" : unchanged
 		}
 
 	</pre>
 </p>
 
 <p>
-	Create a group named <b>Enable video if supported by device</b> and set its weight to <b>130</b>.
+	Create a group named <b>Enable vibration if supported by device</b> and set its weight to <b>120</b>.
```
Comment (antonshkurenko) [inline profilingdemo/resources/projects.xml]: 130 instead of 120 for the vibro

**Review by antonshkurenko — APPROVED** [review]

**Review by Slash32 — APPROVED** [review]

**Pull request merged**